### PR TITLE
Add the ability to change the Tika Server's log file path

### PR DESCRIPTION
**Pull request by imraanparker**
Added a variable to allow the log file for the Tika server to be
different to that of the Jar file path. This is valuable when changing
the TikaJarPath as well. In my case I am putting the Jar file in a
permanent location other than /tmp, but I want the log file to be in
/var/log/ where it can be rotated automatically.

Fixed a duplicate import and some spacing.

### CODE REVIEW

File: tika/tika.py [hunk 1]
```diff
@@ -116,7 +116,7 @@ def die(*s):   warn('Error:',  *s); echo2(USAGE); sys.exit()
 
 def runCommand(cmd, option, urlOrPaths, port, outDir=None, serverHost=ServerHost, tikaServerJar=TikaServerJar, verbose=Verbose, encode=EncodeUtf8):
     """Run the Tika command by calling the Tika server and return results in JSON format (or plain text)."""
-   # import pdb; pdb.set_trace()
+    # import pdb; pdb.set_trace()
```
Comment (chrismattmann) [inline tika/tika.py]: spacing, orthogonal change, please remove.

File: tika/tika.py [hunk 2]
```diff
@@ -142,12 +142,12 @@ def getPaths(urlOrPaths):
     """
     paths = []
     for eachUrlOrPaths in urlOrPaths:
-      if os.path.isdir(eachUrlOrPaths):
```
Comment (chrismattmann) [inline tika/tika.py]: spacing, orthogonal change, please remove.

File: tika/tika.py [hunk 3]
```diff
@@ -158,15 +158,15 @@ def parseAndSave(option, urlOrPaths, outDir=None, serverEndpoint=ServerEndpoint,
...
     paths = getPaths(urlOrPaths)
     for path in paths:
-         if outDir is None:
-             metaPath = path + metaExtension
-         else:
-             metaPath = os.path.join(outDir, os.path.split(path)[1] + metaExtension)
-             echo2('Writing %s' % metaPath)
-             with open(metaPath, 'w', 'utf-8') as f:
-                 f.write(parse1(option, path, serverEndpoint, verbose, tikaServerJar, \
+        if outDir is None:
```
Comment (chrismattmann) [inline tika/tika.py]: spacing, orthogonal change, please remove.

File: tika/tika.py [hunk 4]
```diff
@@ -181,7 +181,7 @@ def parse1(option, urlOrPath, serverEndpoint=ServerEndpoint, verbose=Verbose, ti
           responseMimeType='application/json',
           services={'meta': '/meta', 'text': '/tika', 'all': '/rmeta/text'}):
     """Parse the object and return extracted metadata and/or text in JSON format."""
-    path, type = getRemoteFile(urlOrPath, TikaFilesPath)
+    path, file_type = getRemoteFile(urlOrPath, TikaFilesPath)
```
Comment (chrismattmann) [inline tika/tika.py]: can you explain this change?

File: tika/tika.py [hunk 5]
```diff
@@ -190,7 +190,7 @@ def parse1(option, urlOrPath, serverEndpoint=ServerEndpoint, verbose=Verbose, ti
                                   {'Accept': responseMimeType, 'Content-Disposition': 'attachment; filename=%s' % os.path.basename(path)}, 
                                   verbose, tikaServerJar)
     
-    if type == 'remote': os.unlink(path)
+    if file_type == 'remote': os.unlink(path)
```
Comment (chrismattmann) [inline tika/tika.py]: same here, what's up with this change?

**Comment by chrismattmann** [comment]
Hi @imraanparker  thanks for your contribution! I don't get the spacing updates - why? Also there were some other changes I don't get (type=>file_type) please explain. Thanks.

File: tika/tika.py (hunk 4 above)
Comment (imraanparker) [inline tika/tika.py]: "type" is reserved in python and should be avoided.

File: tika/tika.py (hunk 5 above)
Comment (imraanparker) [inline tika/tika.py]: "type" is reserved in python and should be avoided.

File: tika/tika.py (hunk 1 above)
Comment (imraanparker) [inline tika/tika.py]: Made it PEP 8 compliant to 4 spaces

File: tika/tika.py (hunk 2 above)
Comment (imraanparker) [inline tika/tika.py]: Made it PEP 8 compliant to 4 spaces

File: tika/tika.py (hunk 3 above)
Comment (imraanparker) [inline tika/tika.py]: Made it PEP 8 compliant to 4 spaces

**Comment by imraanparker** [comment]
Hi @chrismattmann. As per my comments above, the spacing updates were so that the code is PEP8 compliant and the other change was that "type" is reserved in python and should be avoided. Python does not complain, but could result in unexpected results.

**Comment by chrismattmann** [comment]
thanks @imraanparker merged! Please feel free to submit another PR to the README.md adding you as a contributor.

**Pull request merged**